test/acceptance/lettuce/integration/terrain.py

```diff
@@ -60,5 +60,5 @@ def after_each_scenario(scenario):
 
 @after.all
 def after_all(total):
-    drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
+    #drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
```

Comment:
Maybe a "Fixme: these steps are comment by {some reason}" could be added above this line?